app/views/bank_accounts/new.js.erb

```diff
@@ -0,0 +1,6 @@
+<% @ajax = true %>
```

Comment:
@dissident it's very bad code. Looks on this code I don't know where this `@ajax` is used. You should directly pass value in partials where it's used.